pkgs/projectcalico/calicoctl/registry.yaml

```diff
@@ -0,0 +1,29 @@
+---
+packages:
+  - name: projectcalico/calicoctl
+    type: github_release
+    repo_owner: projectcalico
+    repo_name: calico
+    asset: calicoctl-{{.OS}}-{{.Arch}}
+    format: raw
+    description: CLI tool for Calico cloud native networking and network security
+    link: https://projectcalico.docs.tigera.io/
+    supported_envs:
+      - darwin
+      - linux
+      - amd64
+      - arm64
+      - armv7
+      - ppc64le
+      - s390x
+    files:
+      - name: calicoctl
+    checksum:
+      type: github_release
+      asset: SHA256SUMS
+      file_format: regexp
+      algorithm: sha256
+      pattern:
+        checksum: ^(\b[A-Fa-f0-9]{64}\b)
+        file: "^\\b[A-Fa-f0-9]{64}\\b\\s+(\\S+)$"
+    rosetta2: true
```

Comment:
```suggestion
```

This setting isn't needed, because the asset for darwin/arm64 [calicoctl-darwin-arm64](https://github.com/projectcalico/calico/releases/download/v3.24.5/calicoctl-darwin-arm64) is released.

https://aquaproj.github.io/docs/reference/registry-config/rosetta2